### DISCUSSION TITLE
HTML BOM integration

### DISCUSSION THREAD

**Discussion by tobiasfalk** [Part-DB/Part-DB-server#815] [Ideas]
I think it would be a nice addition if one could add an HTML BOM(https://github.com/openscopeproject/InteractiveHtmlBom) to a Projekt and have PartDB/the Server host it.

Additional nice to have would be 
- if one could save and reloade a board population in PartDB 
- if one tickes the sourced check box, PartDB automatically removes the number of parts from the system. 

Yes, this requires a more in-depth integration and features on the HTML BOM generator side as well. But it would be really nice.

**Comment by mkne** [comment]
Just for reference: There was the related issue #1150 which got fixed/implemented with release v2.8.0 https://github.com/Part-DB/Part-DB-server/releases/tag/v2.8.0
